### ISSUE TITLE
please allow us to download more than one thing at a time again

### ISSUE THREAD

**Issue by trooper828xx** [Nexus-Mods/Nexus-Mod-Manager#717]
I do not know if this is intentional or not but vortex really seems like an inferior mod loader when i cant download more than one mod at a time, i am fine with being limited to 1 mb/s but it is annoying that i can only download one mod at a time. Please, change to how it used to me.

**Comment by trooper828xx** [comment]
ah i see, it is intentional, having to pay for more than one download. amazing, great, truly a downgrade

**Comment by squid-box** [comment]
While a little hard to follow, it sounds like you have issues with Vortex, and with Nexus' financing strategies. This repository is for technical problems with the (now old and deprecated) Nexus Mod Manager.

I'm closing the issue, but please correct me if I misunderstood.
